src/doing/utils.py

```diff
@@ -168,7 +168,7 @@ def get_config(key: str = "", fallback: Union[str, Dict] = None, envvar_prefix:
 
     # Load the config file
     with open(conf_path) as file:
-        conf = yaml.load(file, Loader=yaml.FullLoader)
+        conf: Dict[str, Any] = yaml.load(file, Loader=yaml.FullLoader)
```

Comment:
Since on L175 we use `get()` on `conf` we sort of "assert" that it is a dict. Normally I would add an assertion here, but since you previously expressed that you favour the dynamicness of Python I opted to just tell the type checker that due to the things we put in `conf_path` this will always be a dict.

I can still add the assertion if you want to.

Reply:
@operte I opted not to use an assertion here. We can do `assert isinstance(conf, dict)` but then you lose the typing of the keys. I don't think it is worth it to add that as well as you'd need to iterate over all keys to do so.

This is probably good enough and helps IDEs with understanding the flow of code.